modules/kubernetes/aks-core/k8s-cluster-role.tf

```diff
@@ -80,3 +80,17 @@ resource "kubernetes_cluster_role" "starboard_reports" {
     verbs      = ["get", "list", "watch", "update", "delete"]
   }
 }
+
+resource "kubernetes_cluster_role" "get-node" {
```

Comment:
Need to snake case `_` resource names :)

Reply:
Fixed :)